packages/happy-css-modules/src/emitter/dts.ts

```diff
@@ -1,23 +1,36 @@
 import { EOL } from 'os';
-import { basename, parse, join } from 'path';
+import path, { basename, parse, join } from 'path';
 import camelcase from 'camelcase';
 import { SourceNode, type CodeWithSourceMap } from '../library/source-map/index.js';
 import { type Token } from '../locator/index.js';
 import { type LocalsConvention } from '../runner.js';
 import { getRelativePath, type DtsFormatOptions } from './index.js';
 
+const CURRENT_WORKING_DIRECTORY = process.cwd();
```

Comment:
It is undesirable to fix the current directory to `process.cwd()`. happy-css-modules can explicitly specify the current directory using the `cwd` option of `runner` function. The `runner` function is exposed as a Node.js API.

- https://github.com/mizdra/happy-css-modules?tab=readme-ov-file#nodejs-api-experimental

Therefore, the `outDir` option must also follow the `cwd` option.

Could you modify your code to allow the following test cases to pass? If this is difficult, I can work on your behalf :)

- https://github.com/mizdra/happy-css-modules/pull/270/commits/047c625d64667da5f7050ea7819c9d3714785839

Reply:
Oh, I see.
I wanted to write a test too but didn't know how to do it, I will modify it to use the cwd option.

Reply:
Thank you.

My advice: [runner.test.ts mocks `console.log` and `console.error`](https://github.com/mizdra/happy-css-modules/blob/047c625d64667da5f7050ea7819c9d3714785839/packages/happy-css-modules/src/runner.test.ts#L26-L29). So if you call `console.log` or `console.error` to debug print, nothing will be output. Use `console.dir` instead.

Reply:
I have tried to fix it and would appreciate a review! https://github.com/mizdra/happy-css-modules/pull/270/commits/d170ef86a46c3ad673099203117f0c86d304ad02